pkg/scanfetcher/nexpose_test.go

```diff
@@ -201,6 +202,26 @@ func TestNexposeClient_FetchScans(t *testing.T) {
 	}
 }
 
+func TestReportingScanFetcher_FetchScans(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	endpoint, _ := url.Parse("http://localhost")
+	timestamp := time.Date(2019, 05, 24, 00, 00, 00, 00, time.UTC)
+
+	nexposeClient := &NexposeClient{
+		Client:   &http.Client{},
```

Comment:
Question: This doesn't appear to be mocking a client, so how is this function being tested?

Reply:
This test was deleted, no longer need to do this :/